components/EventSection.tsx

```diff
@@ -0,0 +1,134 @@
+"use client"
+
+import { useEffect, useState } from "react"
+import { getStringDate } from "@/components/calendar/utils"
+import CalendarEvent from "@/components/calendar/CalendarEvent"
+import CalendarWeekly from "@/components/calendar/CalendarWeekly"
+
+export interface DataProps {
+  id: string
+  date_time: string
+  date: string
+  title: string
+  date_iso: string
+  date_utc: string
+  date2_utc: string
```

Comment:
Why are there so many date fields? Can these be cleaned up?

Also rather than `date2` can we instead do something like `start_date` and `end_date`?

Reply:
This is all fields from the data fetched using LiveWhale API: https://support.livewhale.com/live/blurbs/api#Authenticated-Requests

I need the iso date for accurate calculations (how many mins into the day is this event?), and the utc dates for displaying on the page